crates/neon/src/sys/bindings/mod.rs

```diff
@@ -39,7 +39,7 @@ macro_rules! napi_name {
 ///
 /// Sample input:
 ///
-/// ```ignore
+/// ```skip
```

Comment:
What does `skip` do? I've never seen this one and I couldn't find it in the rustdoc book.